src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -69,24 +88,40 @@ export default function App({ children }) {
   }
 
   const resetSearch = () => {
-    setAppliedStatuses(statusesToFilterBy);
-    setCheckedStatuses(statusesToFilterBy);
+    setAppliedStatusFilters([]);
+    setCheckedStatusFilters([]);
```

Comment:
it looks like this resets and initiates the search with checkboxes that aren't checked. Is this expected behavior?

Reply:
Yeah. Sorry, I should have updated the ticket somewhere with that. [Here's some back-and-forth with Micah regarding that](https://adhoc.slack.com/archives/C05R710CVMH/p1719349406346699?thread_ts=1719244041.433189&cid=C05R710CVMH). 

TL;DR
> No status filters checked = show all claims